packages/nodes-base/nodes/Tributech/descriptions/catalog-api/ManageModelsDescription.ts

```diff
@@ -0,0 +1,137 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const manageModelsOperations = [
+	{
+		displayName: 'Operation',
+		description: 'The operation that should be executed',
+		name: 'operation',
+		type: 'options',
+		displayOptions: {
+			show: {
+				resource: [
+					'manageModels',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Add New Model(s)',
+				description: 'Add New Model(s)',
+				value: 'addNewModels',
+			},
+			{
+				name: 'Get All Entities',
+				description: 'Get All Entities',
+				value: 'getAllEntities',
+			},
+			{
+				name: 'Get Entity',
+				description: 'Get Entity',
+				value: 'getEntity',
+			},
+			{
+				name: 'Revoke Model',
+				description: 'Revoke Model',
+				value: 'revokeModel',
+			},
+		],
+		default: 'addNewModels',
+	},
+] as INodeProperties[];
+
+export const manageModelsFields = [
+	{
```

Comment:
For `Get all`-like operations we always fetch a single page by default, with the maximum number of items in a single page and add a toggle named `Return All` that automatically iterates over all pages and returns all data in a single execution.

Could you please make this change? This makes it consistent with n8n's expected behavior.